modules/ROOT/pages/getting-started-libvirt.adoc

```diff
@@ -32,13 +32,16 @@ virt-install --connect="qemu:///system" --name="${VM_NAME}" --vcpus="${VCPUS}" -
         --os-variant="fedora-coreos-$STREAM" --import --graphics=none \
         --disk="size=${DISK_GB},backing_store=${IMAGE}" \
         --network bridge=virbr0 \
-        --qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=${IGNITION_CONFIG}"
+        --disk path=path/to/example.ign,format=raw,readonly=on,serial=ignition 
 ----
 
 NOTE: `virt-install` requires both the OS image and Ignition file to be specified as absolute paths.
 
 NOTE: Depending on your version of `virt-install`, you may not be able to use `--os-variant=fedora-coreos-*` and will get an error. In this case, you should pick an older Fedora variant (`--os-variant=fedora31` for example). You can find the variants that are supported by you current version of `virt-install` with `osinfo-query os | grep '^\s*fedora'`.
 
+NOTE: In the s390x architecture, for the `startupPolicy` in the disk definition of ignition source file, its optimal to use the default value `mandatory` or `optional`.
+The value `requisite` is allowed only for cdrom or floppy disk type. 
```

Comment:
I think we'd probably want to tell the user to use `optional` to make it non-fatal if the file went away and we'd want to put it in the example. We probably don't need a separate `NOTE:` for it.